src/flux/course.coffee

```diff
@@ -46,7 +47,7 @@ CourseConfig =
     @emit('practice.loaded', obj.id)
 
   _reset: ->
-    CrudConfig.reset.call(@)
+    CrudConfig.reset.call(this)
```

Comment:
Original reason for discouraging empty `@`: https://github.com/jashkenas/coffeescript Issue 1601 (not making it linkable so we are not added to the issue)